airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -119,27 +122,52 @@ public AirbyteCatalog discover(JsonNode config) throws Exception {
     }
   }
 
+  protected void updateSchemaFocus(JsonNode config) {
+    if (config.has("schemaFocus")) {
+      this.schemaFocus = config.get("schemaFocus").asBoolean();
+    } else {
+      this.schemaFocus = false;
+    }
+  }
+
+  protected boolean getSchemaFocus() {
+    return this.schemaFocus;
+  }
+
+  protected List<String> getExcludedInternalSchemas() {
```

Comment:
it would be nice if we could get jooq to do this for us. i know you're not super familiar with jooq, so if it's not obvious how to do this, we can skip it for now and just keep what you've got. i would do a really quick glance to see if in jooq you can get `context.meta().getSchemas(databaseName);` to just return the non internal tables.

Reply:
any follow up on this?

Reply:
looks like you only ever override this for MySql. Do MSSQL and Postgres not have this problem? Do we need to do this override manually for every database or just the ones that aren't compatible with jooq?

Reply:
I haven't found anything in jooq to do this yet and no it doesn't seem like the other databases need to manually exclude anything...

Reply:
I'm using it for the RedshiftSource too:
https://github.com/airbytehq/airbyte/pull/1009/files

Reply:
okay. thanks! approved.